regolith/schemas.py

```diff
@@ -1615,6 +1658,102 @@
             "type": "string",
         },
     },
+    "expenses": {
+        "_description": {
+            "description": "This collection records expenses for the "
+            "group. It should most likely be private"
+        },
+        "_id": {
+            "description": "short representation, such as this-is-my-name",
+            "required": True,
+            "type": "string",
+        },
+        "project": {
+            "description": "project or list of projects that this "
+            "presentation is associated with.  Should "
+            "be discoverable in projects collection",
+            "required": True,
+            "anyof_type": ["string", "list"],
+        },
+        "payee": {
+            "description": "The name or id of the payee filing the expense",
+            "required": True,
+            "type": "string",
+        },
+        "itemized_expenses": {
+            "type": "list",
+            "schema": {
+                "type": "dict",
+                "schema": {
+                    "day": {
+                        "description": "Expense day",
+                        "required": True,
+                        "type": "integer",
+                    },
+                    "month": {
```

Comment:
I think at the end of the day we should accept whatever format for months people want to provide and have some tool(s) which to the conversion for the various builders. That way we are move accepting of inputs and make sure that everything comes out as needed for the report.

Reply:
I agree with this.  The problem was in the preslist builder I didn't just want to typeset the date, I wanted to evaluate it so that I could sort by date.  I  think that is now possible?  If so, we can change the preslist builder and make everything consistent.  That is my point, but @CJ-Wright please confirm that this is the case.   btw, this doesn't affect this PR.

Reply:
@CJ-Wright , Done, new issue created.